pages/nft-book-store/status/[classId].vue

```diff
@@ -9,16 +9,45 @@
     </div>
     <hr>
     <section v-if="bookStoreApiStore.isAuthenticated">
+      <h3>Editions</h3>
+      <table>
+        <thead>
+          <tr>
+            <th>Order</th>
+            <th>Name</th>
+            <th>Price (USD)</th>
+            <th v-if="userIsOwner">Sort</th>
+          </tr>
+        </thead>
+        <Draggable
+          v-model="prices"
+          tag="tbody"
+          item-key="index"
+          handle="td:last-child"
+          :disabled="!userIsOwner || isUpdatingPricesOrder"
```

Comment:
Do we allow moderator to update this field?

Reply:
Not in the requirement, let me confirm